corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_properties.html

```diff
@@ -65,8 +65,34 @@ <h3>
                       <label class="control-label">{% trans "Sort Field" %}</label>
                       <input class="form-control" type="text" data-bind="value: property.itemSet.sort"/>
                     </div>
+                    <div class="panel panel-default">
+                      <div class="panel-heading">
+                        <h4 class="panel-title">
+                          {% trans "Advanced" %}
```

Comment:
The spacing on this in the screenshot is strangely cramped, I think because this is an `h4` not an `h3`: https://getbootstrap.com/docs/3.3/components/#panels-heading

Reply:
It actually needed `panel-title-nolink` as a class, since we override the regular `panel-title` somewhere